components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/role-mgt.jsp

```diff
@@ -385,7 +431,92 @@
+                             <td>
+                                 <%if(data.getDomainName() != null){%>
+                                 <%data.getDomainName();%>
+                                 <%} %>
+                             </td>
+                         <%}%>--%>
+                    <td>
+                        <%if (!flaggedName.getShared()) { %>
+                        <% if (flaggedName.getItemName().equals(userRealmInfo.getAdminRole()) == false && flaggedName.getItemName().equals(userRealmInfo.getEveryOneRole()) == false && flaggedName.getEditable()) {%>
+                        <a href="#" onclick="updateUserGroup('<%=roleName%>')" class="icon-link"
+                           style="background-image:url(images/edit.gif);"><fmt:message key="rename"/></a>
+                        <% } %>
+                        <% if (!flaggedName.getItemName().equals(userRealmInfo.getAdminRole())) {%>
+                        <a href="edit-permissions.jsp?roleName=<%=roleName%>" class="icon-link"
+                           style="background-image:url(images/edit.gif);"><fmt:message key="edit.permissions"/></a>
+                        <% }
+                        }%>
+
+                        <% if (!userRealmInfo.getEveryOneRole().equals(flaggedName.getItemName()) && flaggedName.getEditable()) { %>
+                        <a href="edit-users.jsp?roleName=<%=roleName%>&<%=UserAdminUIConstants.ROLE_READ_ONLY%>=<%=!flaggedName.getEditable()%>"
+                           class="icon-link" style="background-image:url(images/edit.gif);"><fmt:message
+                                key="edit.users"/></a>
+                        <% } %>
+                        <% if (!userRealmInfo.getEveryOneRole().equals(flaggedName.getItemName())) { %>
+                        <a href="view-users.jsp?roleName=<%=roleName%>&<%=UserAdminUIConstants.ROLE_READ_ONLY%>=<%=!flaggedName.getEditable()%>"
+                           class="icon-link" style="background-image:url(images/view.gif);"><fmt:message
+                                key="view.users"/></a>
+                        <% } %>
+
+                    </td>
+                </tr>
+
+                <%
+                            }
+                        }
+                    }
+                %>
+
+                <%
+                    if (workFlowAddPendingRoles != null) {
+                        for (Iterator<String> iterator = workFlowAddPendingRoles.iterator(); iterator.hasNext(); ) {
+                            String role = iterator.next();
+                            if (role != null) { //Confusing!!. Sometimes a null object comes. Maybe a bug Axis!!
+                                if (role.equals(CarbonConstants.REGISTRY_ANONNYMOUS_USERNAME)) {
+                                    continue;
+                                }
+                                String disPlayName = role + " " + "[Pending Roles for Add]";
```

Comment:
Concat the two Strings.
The suffix should be changed as "Pending to be added" ?